packages/server-dev/src/manager/processes/nextBuild.mjs

```diff
@@ -16,14 +16,14 @@
 
 import { spawnProcess } from '@lowdefy/node-utils';
 
-function nextBuild({ packageManager, verbose }) {
+function nextBuild(context) {
   return async () => {
     console.log('Building app...');
```

Comment:
Add `TODO` to pass down logger to these methods to use and pass through to spawnProcess.